nifi-nar-bundles/nifi-hazelcast-bundle/nifi-hazelcast-services/src/main/resources/docs/org.apache.nifi.hazelcast.services.cachemanager.EmbeddedHazelcastCacheManager/additionalDetails.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<head>
+    <meta charset="utf-8" />
+    <title>EmbeddedHazelcastCacheManager</title>
+    <link rel="stylesheet" href="/nifi-docs/css/component-usage.css" type="text/css" />
+</head>
+
+<>
+<h2>EmbeddedHazelcastCacheManager</h2>
+
+<p>
+    This service starts and manages an embedded Hazelcast instance. The cache manager has direct accesses to the
+    instance - and the data stored in it. However, the instance sill opens a port for potential clients to join and
+    this cannot be prevented. Note that this might leave the instance open for rogue clients to join.
+</p>
+
+<p>
+    It is possible to have multiple independent Hazelcast instances on the same host (whether via EmbeddedHazelcastCacheManager
+    or externally) without any interference by setting the properties accordingly. If there are no other instances, the port number
+    and cluster name are not necessary to be set. (Default values will be used instead.)
+</p>
+
+<p>
+    The service supports multiple ways to set up a Hazelcast cluster. This is controlled by the property, named "Hazelcast High
+    Availability Mode". The following values might be used:
```

Comment:
I can agree on using a more describing name for "Hazelcast High Availability Mode" but Cluster _Mode_ sounds like it would describe the way the connect not the way the members are determined. I think Hazelcast Clustering Strategy might work out.

I think, you are right with the All Nodes as in case it's available are instances are nodes. As for None, I think the new name of the property changes the meaning, thus I find it describing. I might be "standalone" or "local" but these comes with secondary meaning would suggest misleading things.

None is the default mode as it's the only one valid with both clustered and standalone NiFi instance.